eth2util/keystore/keystore.go

```diff
@@ -55,48 +61,62 @@ func StoreKeys(secrets []tbls.PrivateKey, dir string) error {
 }
 
 func storeKeysInternal(secrets []tbls.PrivateKey, dir string, filenameFmt string, opts ...keystorev4.Option) error {
```

Comment:
can we add a test to store/load 100 keys?

Reply:
Will add one in a separate PR as discussed offline.